flixel/system/frontEnds/DebuggerFrontEnd.hx

```diff
@@ -168,6 +168,18 @@ class DebuggerFrontEnd
 			FlxG.stage.focus = null;
 			// setting focus to null will trigger a focus lost event, let's undo that
 			FlxG.game.onFocus(null);
+
+			#if FLX_MOUSE
+			FlxG.mouse.enabled = true;
+			#end
+		}
+		else
+		{
+			#if FLX_MOUSE
+			// Debugger is visible, allow mouse input in the game only if the
+			// interaction tool is not active.
+			FlxG.mouse.enabled = !FlxG.game.debugger.interaction.visible;
```

Comment:
This is a bit annoying - just because it's visible doesn't mean it's actively being used. I'd like to still be able to interact with buttons as long as none of the interaction tools are selected.

Reply:
I agree. I'll change that.